ext/openssl/xp_ssl.c

```diff
@@ -948,15 +950,34 @@ static int php_openssl_set_local_cert(SSL_CTX *ctx, php_stream *stream) /* {{{ *
 					certfile);
 				return FAILURE;
 			}
+			ctx_set = 1;
+		/* val is still local_cert/certfile since GET_VER_OPT_STRING("local_cert", certfile) */
+		} else if ( (cert = php_openssl_x509_from_str(Z_STR_P(val))) != NULL ) {
```

Comment:
Slight style issue - please don't add space after `(` and before `)`

Reply:
ok

Reply:
done https://github.com/php/php-src/pull/6860/commits/747e0658bcf54ad6c00b8b0fb9e77ee67dfa8f3c